### ISSUE TITLE
Issues upgrading to 0.5.0 from 0.2.0

### ISSUE THREAD

**Issue by nmiculinic** [timescale/timescaledb#255]
I'm using dockerized setup for timescaledb. I run it as: 

```docker run -d --restart=always -v /mnt/db/pgdata:/var/lib/postgresql/data -p 5432:5432 timescale/timescaledb:0.5.0``` 

and in such way my data is preserved.  I run 0.2.0 version the same way. Upon using ```docker inspect``` and observing the mountpoints I see expected output:
```
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/mnt/db/pgdata",
                "Destination": "/var/lib/postgresql/data",
                "Mode": "",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
```

Recently I tried migrating from 0.2.0, to 0.5.0. Following the instuctions I did:
```
docker exec -it db psql -U postgres
postgres=# ALTER EXTENSION timescaledb UPDATE;
ERROR:  extension "timescaledb" does not exist
postgres=# \dx
                 List of installed extensions
  Name   | Version |   Schema   |         Description          
---------+---------+------------+------------------------------
 plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
(1 row)
```

And as you can see this isn't the output I wished to see. What happened, and how can I fix it?

**Comment by mfreed** [comment]
Hi @nmiculinic :  It's a bit hard to debug just from the above information.  The error from `ALTER EXTENSION` just seems to suggest that postgres isn't finding the extension installed on the database you are trying to update.

One thing to note is that you actually install/extensions in individual *databases*, not the database *instance*.  From the psql command line, that's `psql -d <database_name>` if it's different from the default.  I can't quite tell from just your run command if you actually first installed timescale into a different database.  Can you clarify this?

Basically, if you run `ALTER EXTENSION`, you should first see the extension (via `\dx`) before you try to upgrade it.

**Comment by nmiculinic** [comment]
I've tried with -d <database name> and it seems to be working. Though before the upgrade I got: 
```

db=# \dx
ERROR:  Oid lookup failed for table index chunk_constraint_chunk_id_dimension_slice_id_idx
db=# ALTER EXTENSION timescaledb UPDATE;
ALTER EXTENSION
db=# \dx
                                      List of installed extensions
    Name     | Version |   Schema   |                            Description                            
-------------+---------+------------+-------------------------------------------------------------------
 plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
 timescaledb | 0.5.0   | public     | Enables scalable inserts and complex queries for time-series data
(2 rows)

db=# 


```

**Comment by RobAtticus** [comment]
That's to be expected. When you update timescale, it installs a new library file (`timescaledb.so`) that is not compatible with old versions of the extension. So when you first connect to a database and then run almost any command, you'll see errors like that. However once you run ALTER EXTENSION, the `.so` file and database extension are now in sync so it works again.

We are working on a fix to make this a little nicer for future upgrades.

**Comment by nmiculinic** [comment]
Thank you!
